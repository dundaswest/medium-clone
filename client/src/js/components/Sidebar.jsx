import React from 'react';
import clap from '../clap.svg';
import twitter from '../twitter.svg';
import bookmark from '../bookmark-white.svg';
import facebook from '../facebook.svg';
import store from '../redux/store/store';
/*
class Sidebar extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div className="sideBar">
        <div>
          <div id="clapNum">
            300
          </div>
        <img src={clap} id="clap" />
        </div>
        <div>
        <img src={twitter} id="twitter" />
        </div>
        <div>
        <img src={bookmark} id="bookmark" />
        </div>
        <div>
        <img src={facebook} id="facebook" />
        </div>

      </div>
    );
  }
}
*/
const Sidebar = ({clap_num, onClick }) => (
  <div className="sideBar">
  <div>
    <div id="clapNum">
      {clap_num}
    </div>
  <img src={clap} id="clap" onClick = {onClick}/>
  </div>
  <div>
  <img src={twitter} id="twitter"/>
  </div>
  <div>
  <img src={bookmark} id="bookmark" />
  </div>
  <div>
  <img src={facebook} id="facebook" />
  </div>

</div>
)
export default Sidebar;
