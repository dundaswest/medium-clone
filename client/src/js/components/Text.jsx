import React from 'react';
import DefaultText from './DefaultText';

const Text = () => (
  <div className="TextDiv">
    <DefaultText />
  </div>
);

export default Text;
