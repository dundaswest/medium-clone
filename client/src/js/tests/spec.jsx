import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, configure } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import Sidebar from '../components/Sidebar';
import Text from '../components/Text';

configure({ adapter: new Adapter() });

test('Side bar renders correctly', () => {
  const component = shallow(<Text />);
  // let tree = component.toJSON();
  expect(component).toMatchSnapshot();
});
