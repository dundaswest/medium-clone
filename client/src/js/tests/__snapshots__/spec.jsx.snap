// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Side bar renders correctly 1`] = `
<div
  className="TextDiv"
>
  <div
    className="ArticleText"
  >
    Lorem ipsum dolor sit amet, no cum unum epicuri! Nam an mazim nihil necessitatibus, sit in mundi semper definitionem? Minim melius saperet ut usu? Mea mundi adolescens contentiones ex, eros assueverit deterruisset id est? Ne ceteros appetere forensibus ius, ullum indoctum duo ut, ius ea sonet epicurei reprimique. Verear accommodare nam ex! An eam quis omnes, usu ne maiorum intellegam, ad vix nulla commodo delicata. Alia diceret in cum, illud error quaestio per te? Cum magna facilisis omittantur eu, ex decore putent mei, cu quem laboramus est. In eros mandamus pri, vocent accusam sit at, id eos splendide dissentiet. Te cum sale atqui clita, per cu magna aeterno. Nonumes scaevola intellegebat est ei, ne quem utroque persequeris sit. Ex munere bonorum disputando nec, eos exerci nominati cu. Adhuc intellegebat eu vel. Intellegat scriptorem disputationi ut nec. Cu sed zril mediocrem neglegentur, his at putent repudiare quaerendum, iracundia vituperatoribus ei sit? Munere nostro mea ea? In vix labitur civibus, vis zril consul iracundia at. Tollit eligendi sed in. Cu quo augue invidunt moderatius, est in augue ocurreret sadipscing, usu voluptua repudiandae cu. Eum no aliquam quaerendum intellegebat. Ne vix quaestio deseruisse scripserit! Pericula incorrupte est ad, meliore sadipscing pro ex, an usu elit vocent facilisis! Te sea illum instructior, rebum nihil pro in. Ea vel ubique probatus temporibus. Mundi fabellas eum no. An sed bonorum perpetua persequeris, natum eripuit qualisque sit at. Vitae officiis definitiones in pro. Mei prompta molestie lobortis at, purto minimum et est. Ea veniam numquam appareat eum. Ludus nominati senserit cu eos, his probo epicuri ex! Mei ei tempor possit, has eu dicam dolore hendrerit? Ad quaeque sensibus philosophia sed. Discere reprehendunt ad sit, primis delenit officiis an sit. Cu fugit nostrum consequuntur eam, scaevola efficiantur disputationi his et, feugiat constituam te eum. Id quo justo exerci, eu vis veritus accusata. Movet alterum admodum sea ex, an partem cetero complectitur est. Eos te choro theophrastus. Eu oporteat mandamus corrumpit mel? Id mei rebum volumus sadipscing, vis at quas omnium numquam. Sed augue tantas dicunt ad, eu eos diceret patrioque abhorreant! Mel ne stet altera, mutat philosophia in has, ad sale sonet consul his! Alii dicunt qui et, ius adhuc blandit id. Cu eam ignota labores, qui ei agam adolescens. Eu habeo solet signiferumque per. Veniam bonorum ei sit, no omnes incorrupte intellegam pro? Fugit epicurei corrumpit ea eam, omnis persius mea ne. Has suscipit appetere an, esse munere viderer cu sea. Ex est unum exerci. Mea novum exerci consequat cu, eum ne error nullam! Utamur detraxit assueverit nec te, pro ad suas congue delectus. Ad eos diceret insolens! Nam perpetua similique et, at est debitis voluptua. Ei eum harum postulant, ne mel simul bonorum? Cu altera persius partiendo sea, ne eum paulo praesent, his eu graece adipiscing. Oratio tibique no has, ea esse deserunt vituperata pro. Sale ignota reprimique pro ea! Ut legere oblique eum, ad laudem gubergren percipitur eos, ei sale offendit sit! Ea quidam nemore sapientem vel, vix cu epicurei menandri principes. In vis dolor inermis. Eum cu rebum ceteros habemus, id illum quando option eam! Ut iudicabit urbanitas cum! Idque eligendi legendos te ius, te minim nostrum democritum eos? Te has invenire honestatis quaerendum, eam tritani fastidii te. No cum percipit scriptorem. Ut laudem aperiam suavitate vim! Quo et tale fierent! Eam rebum primis quaeque et. No eam consul dictas vulputate, mei choro primis eligendi ex, erat platonem ius an. Et iisque legendos pertinax qui, homero aperiam et sed. An mundi laudem everti nam, no veniam menandri nam? Sea et scaevola conceptam. Vel eu labores lobortis temporibus? Vis meliore mnesarchum ex, error aeterno his at. Vidit mazim vim ex, quo eirmod oblique offendit in, usu eu lorem perfecto. Deseruisse efficiendi definitionem eum at, eum an postea gloriatur, ea eos dolore dissentiet? Vero doming adolescens vis no? Sed ei deserunt liberavisse, modus corpora corrumpit nam ei, oportere volutpat evertitur in mei. Primis sapientem deseruisse nam eu, vis cu utamur facilis quaerendum. Ne sea legimus accusata aliquando, est iudico iisque ei. In per assum noluisse prodesset, utroque convenire sit in, agam detraxit rationibus mei et. Eos at facete ocurreret splendide, cum quidam integre impedit id. Dicta homero iisque eu nam, dicunt conclusionemque ut vim, eu vel oratio consul? Vel stet etiam ei, cu dolore putant appellantur vix. Ex maiorum accusam qui, principes definitionem cu vix, per an aliquip honestatis reformidans. Vix meis dolores delicata id. Eam soleat essent in. Ea eam debet nominati. Sonet quaestio vis eu, dolor essent epicuri sea no. Sale quidam repudiandae sed at, no vim commune detraxit! Nec quodsi impedit persecuti eu, ne perfecto consequuntur his! Sea ex essent accusam, omnis aeterno mel ut, elaboraret reformidans vix ex? Eu agam graecis definiebas has, eum at quem moderatius. Eos ne mutat tamquam. Malis etiam adipiscing id pri! Eu mea primis convenire. Petentium consequuntur per ad, ut ignota diceret nam, nullam ubique delicata vim id? Modus fuisset ex eos, mei movet impetus id. Debet detraxit eleifend mei eu? Eu duo quaeque fabulas laboramus, debet adolescens eam et. Graeci veritus at mei, has evertitur vituperata et. Vim ex legimus laoreet praesent, id vis feugait atomorum! Te ferri dolores cum, impetus adipisci no sea! Id his velit salutatus delicatissimi? Ea nusquam disputando nec, veri porro usu te. Vis te nonumy vocent adolescens, mnesarchum scribentur ne mea? Mundi commodo virtute no pri, vel dolorum antiopam voluptatum in. Dicat putent persecuti cu usu. No usu omnesque instructior! Vim dicit expetendis an! Pri meis animal denique cu? Ei sea unum intellegam. Placerat elaboraret pro ea? Vitae admodum neglegentur ei nam? Eam facete iisque ei, amet accusata philosophia ei nam? Usu in noster recusabo, id duo commodo ceteros, suscipit inciderint vim ut? Dolore mediocrem repudiare an eam? In eos solet iracundia, his in audiam denique mediocrem. Vix eu sanctus dignissim adipiscing. Odio possit abhorreant pro in. Ridens complectitur eum ad, an euismod mentitum constituto sea. Eos cu appareat torquatos, recteque liberavisse per te, ne mea detracto periculis. Idque constituto eu nec, sed ea placerat posidonium. Ea maiorum tibique euripidis has.
  </div>
</div>
`;
